src/ports/postgres/modules/svm/svm.py_in

```diff
@@ -144,30 +145,37 @@ def _verify_get_params_dict(params_dict):
 
 
 def _build_output_tables(n_iters_run, model_table, args, transformer, **kwargs):
-    if transformer is None:
+    # transformer should always be a valid object
+    original_table = transformer.original_table
+    if not original_table:
```

Comment:
I agree we should add a note to ensure that future kernels never set it to empty. It also makes sense that `original_table` should always be non-empty otherwise there is nothing to train from.